src/data-scraper/index.ts

```diff
@@ -101,8 +101,10 @@ export async function scrapeData(): Promise<void> {
         2,
       );
 
-      logger.info({}, 'sending runtime data upstream');
```

Comment:
should we have a log with something about no runtime results retrieved? It is good to be explicit in this component because it runs on a customers cluster

Reply:
Well we did receive them, but we just filtered them all out - it's not surprising given that we only pull 10 runtime images at a time that sometimes we're not interested in any of them.